system_test/common/helpers/aest_docker.erl

```diff
@@ -223,19 +223,46 @@ setup_node(Spec, BackendState) ->
         end,
     HardForkVars =
         case maps:find(hard_forks, Spec) of
-            error -> #{};
             {ok, HardForks} ->
-                #{hard_forks_present => [#{}],
-                  hard_forks =>
-                      lists:map(fun({V, H}) -> #{version => V, height => H} end,
-                                maps:to_list(HardForks))}
+                #{hard_forks =>
+                      #{present => true,
+                        hard_fork_info =>
+                            lists:map(
+                              fun({V, H}) -> #{version => V, height => H} end,
+                              maps:to_list(HardForks))}};
+            error ->
+                #{hard_forks =>
+                      #{present => false,
+                        hard_fork_info => []}}
+        end,
+    ForkManagementVars =
+        case maps:find(fork_management, Spec) of
+            {ok, ForkManagement} ->
+                Fork = maps:get(fork, ForkManagement),
+                SigStartHeight = maps:get(signalling_start_height, Fork),
+                SigEndHeight = maps:get(signalling_end_height, Fork),
+                SigBlockCount = maps:get(signalling_block_count, Fork),
+                ForkHeight = maps:get(fork_height, Fork),
+                InfoField = maps:get(info_field, Fork),
+                Version = maps:get(version, Fork),
+                #{fork_management =>
+                      #{fork =>
+                            #{signalling_start_height => SigStartHeight,
+                              signalling_end_height => SigEndHeight,
+                              signalling_block_count => SigBlockCount,
+                              fork_height => ForkHeight,
+                              info_field => InfoField,
+                              version => Version}}};
+            error ->
+                #{fork_management => false}
         end,
-    RootVars = (maps:merge(CuckooMinerVars, HardForkVars))#{
+    RootVars = maps:merge(CuckooMinerVars, HardForkVars),
+    RootVars1 = maps:merge(RootVars, ForkManagementVars),
+    RootVars2 = RootVars1#{
         hostname => Name,
         ext_addr => format("http://~s:~w/", [Hostname, ?EXT_HTTP_PORT]),
         peers => PeerVars,
         key_password => ?PEER_KEYS_PASSWORD,
-        config => ExtraConfig,
```

Comment:
???
So now this variable is bound (from key `config` documented in type spec) but unused.

Not sure why fork-specific handling - rather than re-using this ExtraConfig - though there must be reason so ok...